pkg/grpc/client.go

```diff
@@ -15,6 +16,8 @@ import (
 	"github.com/pomerium/pomerium/pkg/telemetry/requestid"
 )
 
+var OutboundAddress = filepath.Join(os.TempDir(), "pomerium-outbound.sock")
```

Comment:
Possibly we should use `$XDG_RUNTIME_DIR` if its available?

Reply:
The test environment uses/modifies $TMPDIR (picked up by os.TempDir()) to separate extracted files for individual envoy instances, so if we put the socket somewhere else by default we'd also need a way to override that